docs/plat/rpi3.rst

```diff
@@ -237,6 +237,13 @@ The following build options are supported:
   in AArch64 mode. If set to 1, it will jump to BL33 in Hypervisor in AArch32
   mode.
 
+- ``BL32``: This port can load and run OP-TEE image.
+  OP-TEE image is optional. Please use the code from the following link
+  `link <https://github.com/OP-TEE/optee_os>`__
+  Use BL32=tee-pager.bin or BL32=tee-header_v2.bin
+  BL32_EXTRA1=tee-pager_v2.bin BL32_EXTRA2=tee-pageable_v2.bin
+  to put optee images into FIP image.
```

Comment:
```
- ``BL32``: This port can load and run OP-TEE. The OP-TEE image is optional.
  Please use the code from `here <https://github.com/OP-TEE/optee_os>`__. Build
  the Trusted Firmware with option ``BL32=tee-pager.bin`` or
  ``BL32=tee-header_v2.bin BL32_EXTRA1=tee-pager_v2.bin
  BL32_EXTRA2=tee-pageable_v2.bin`` to put the binaries into the FIP.
```

Reply:
Anyway, I was referring to the diagram in line 100. Just write BL32 like with the other images.